Llk/Llk.php

```diff
@@ -125,7 +125,7 @@ public static function save(Parser $parser, $className)
                     '                    \'' . $tokenName . '\' => \'' .
                     str_replace(
                         ['\'', '\\\\'],
-                        ['\\\'', '\\\\\\'],
+                        ['\\\'', '\\\\\\\\'],
```

Comment:
It seems like dumping of these values must be done via `var_export(..., true)`. Right now it's unsafe and unpredictable.

Reply:
let's fix the bug within the current mechanism first because it's an easy merge. You can open a PR where var_export is used

Reply:
Actually, using `var_export` is not a bad idea at all. And it will simplify the code. Are you willing to try to update with `var_export` please?